qtpy/tests/test_qtgui.py

```diff
@@ -61,6 +61,39 @@ def test_enum_access():
     assert QtGui.QIcon.Normal == QtGui.QIcon.Mode.Normal
     assert QtGui.QImage.Format_Invalid == QtGui.QImage.Format.Format_Invalid
 
+
+@pytest.mark.skipif(
+    sys.platform.startswith('linux') and not_using_conda(),
+    reason="Fatal Python error: Aborted on Linux CI when not using conda")
+@pytest.mark.skipif(
+    sys.platform == 'darwin' and sys.version_info[:2] == (3, 7),
+    reason="Stalls on macOS CI with Python 3.7")
+def test_QMouseEvent_pos_functions(qtbot):
+    """Test `QMouseEvent.pos` and related functions obsolete in Qt6,
+       test `QMouseEvent.position` and related functions missing from Qt5"""
```

Comment:
Minor nits, but per the docstring standard and with some minor fixes, this should be

```suggestion
    """
    Test `QMouseEvent.pos` and related functions removed in Qt 6,
    and `QMouseEvent.position`, etc., missing from Qt 5.
    """
```